source/backend/api/Services/LeaseService.cs

```diff
@@ -64,6 +71,26 @@ public PimsLease Add(PimsLease lease, bool userOverride = false)
 
         public PimsLease Update(PimsLease lease, bool userOverride = false)
         {
+            var currentLease = _leaseRepository.Get(lease.LeaseId);
+
+            if (currentLease.LeaseStatusTypeCode != lease.LeaseStatusTypeCode)
+            {
+                _entityNoteRepository.Add<PimsLeaseNote>(
+                    new PimsLeaseNote()
+                    {
+                        LeaseId = currentLease.LeaseId,
+                        AppCreateTimestamp = System.DateTime.Now,
```

Comment:
is this necessary? I know we do some automatic insertion of these audit values in most scenarios. Not sure if that is applicable here.